src/upload-course-content.js

```diff
@@ -94,12 +95,17 @@ module.exports = class UploadCourseContent {
 
 	async _createModule(instanceUrl, orgUnit, module, parentModule) {
 		const url = parentModule
-			? new URL(`/d2l/api/le/1.34/${orgUnit.Id}/content/modules/${parentModule.Id}/structure/`, instanceUrl)
-			: new URL(`/d2l/api/le/1.34/${orgUnit.Id}/content/root/`, instanceUrl);
+			? new URL(`/d2l/api/le/1.34/${orgUnit.Identifier}/content/modules/${parentModule.ModuleId}/structure/`, instanceUrl)
```

Comment:
I think `parentModule.Id` was correct:
https://docs.valence.desire2learn.com/res/content.html#Content.ContentObject

Reply:
Ah - I was looking at the TOC. Thanks for catching that - will fix.